Library/Homebrew/cmd/update-report.rb

```diff
@@ -422,6 +426,7 @@ def dump
     dump_formula_report :M, "Updated Formulae"
     dump_formula_report :R, "Renamed Formulae"
     dump_formula_report :D, "Deleted Formulae"
+    dump_formula_report :MC, "Updated Casks"
```

Comment:
```suggestion
    dump_formula_report :MC, "Updated Casks"
    dump_formula_report :DC, "Deleted Casks"
```

as I noticed it's already populating that array.